### PR TITLE
Adds suggested catchables from doobie

### PR DESCRIPTION
**Pull request by skeet70**
For issue #12, still need to add examples to the documentation. Thanks for the suggestion @tpolecat.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,4 +267,4 @@ For those with permission to release:
     * Mac users can `brew install gpg pinentry-mac` to get the tools needed.
 * Create a Sonatype [credentials file](http://www.scala-sbt.org/1.0/docs/Using-Sonatype.html#Fourth+-+Adding+credentials).
 * Run `sbt release`
-* Push the newly created tags and version bump commits to `rubicon-project/rubiz`. 
+* `git push` to add the new tags and release commits to master.
```
Comment (coltfred) [inline README.md]: This is unintentional.

Comment (skeet70) [reply README.md]: Finally made that change.

**Comment by skeet70** [comment]
@coltfred think I fixed the README issue for realsies.

**Comment by leifwickland** [comment]
LGTM 👍

**Review by leifwickland — APPROVED** [review]

**Comment by coltfred** [comment]
Looks fine, though I'd take out the 

```
+Example needed.
```
if you're not going to put examples in there. You could just link them to the tests if you don't want to bother putting the examples in tut.

**Comment by skeet70** [comment]
@coltfred I kinda wanted to add those in this PR but haven't gotten to it. PRs to my PR welcome :D

**Comment by skeet70** [comment]
@coltfred code coverage is telling me `onException` and `ensuring` aren't covered, but it seems like they are to me.

**Comment by coltfred** [comment]
I think I got you squared on the code coverage, right?

**Comment by skeet70** [comment]
Yeah, coverage is back up to 100%. Got the one that @leifwickland missed too. Should have examples for everything now, so I'm going to merge. If you've got beef with my examples I can open a new PR later.

**Pull request merged**